README.md

```diff
@@ -47,7 +47,8 @@ import napari
 viewer = napari.view_image(data.cells3d(), channel_axis=1, ndisplay=3)
 ```
 
-![napari viewer showing an image of an astronaut.](https://github.com/napari/docs/blob/main/docs/images/screenshot-add-image.png)
+<img width="1198" alt="napari viewer showing cells in 3d." src="https://user-images.githubusercontent.com/44469195/209934762-aba0c2c0-5829-43ac-96f1-ce55707860b4.png">
```

Comment:
```suggestion
![napari viewer with a multichannel image of cells displayed as two image layers: nuclei and membrane.](https://github.com/napari/docs/blob/main/docs/images/multichannel_cells.png)
```
I think this is the preferred way of fixing this.

Reply:
sounds good.
I just want to note that now the generated image has other parameters than given from the readme example code.
 The actual code that generates this new example is here: 
https://napari.org/stable/howtos/layers/image.html#loading-multichannel-images

Reply:
You're right of course. Then again looking more closely, the code block on that page also doesn't produce that image, because it's not making the viewer 3D.
But yeah, the contrast limits are changed to make it a bit contrast-ier and it's rotated to show the 3D.

Reply:
Anyhow, for me it's fine. Maybe @jni can take a look and make the final call whether we go with that image (which is also on napari.org) or make a new one or use yours (but add it to the napari/docs repo).

Reply:
> contrast limits are changed to make it a bit contrast-ier and it's rotated to show the 3D.  

I think it's great that it looks shiny with these tweaked parameters, and at the same it's good that the example code is kept at the bare minimum. This PR can be merged from my side as well :)